vault/capabilities.go

```diff
@@ -58,14 +58,29 @@ func (c *Core) Capabilities(ctx context.Context, token, path string) ([]string,
 		policyCount += len(nsPolicies)
 	}
 
+	// Construct the corresponding ACL object. ACL construction should be
+	// performed on the token's namespace.
+	tokenCtx := namespace.ContextWithNamespace(ctx, tokenNS)
+	policies, err := c.policyStore.GetNamedPolicies(tokenCtx, policyNames)
```

Comment:
It looks like whenever `InlinePolicies` are involved, we have to call `GetNamedPolicies`, `ParseACLPolicy`, followed by `ACLFromPolicies`. Is there a way to wrap/simplify this?

Reply:
Good idea. Simplified the handling in 9af58c4.